setup.py

```diff
@@ -18,7 +18,9 @@
 
 REQUIRES = [
     "tiledb>=0.5.0",
-    "urllib3>=1.26",
+    # One of our upstream dependencies needs urllib<1.26.
+    # This should be relaxed once that is fixed.
+    "urllib3==1.25.11",
```

Comment:
Should this be a wildcard on `1.25` or do we need the specific `1.25.11`?

Reply:
Possibly, but I don't think they will be coming out with any more 1.25s and this is (hopefully) a temporary measure.